pkg/network/ebpf/c/shared-libraries/types.h

```diff
@@ -1,5 +1,5 @@
-#ifndef __SOWATCHER_TYPES_H
-#define __SOWATCHER_TYPES_H
+#ifndef __SHARED_LIBRARIES_H
+#define __SHARED_LIBRARIES_H
```

Comment:
Please have the ifdef names match the filename/context. Maybe `__SHARED_LIBRARIES_TYPES_H`?